.github/workflows/test.yaml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     paths:
       - '**/*'
+  workflow_call:
```

Comment:
Needed to make this workflow reusable ([GitHub docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow))